portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -3,11 +3,37 @@
 /** An item on a todo list. */
 public final class Comment {
 
-  private final String comment;
-  private final long timestamp;
+    private final String name;
+    private final String comment;
+    private final long timestamp;
+    private final String mood;
+    private final long id;
 
-  public Comment(String comment, long timestamp) {
-    this.comment = comment;
-    this.timestamp = timestamp;
-  }
+    public Comment(String name, String comment, long timestamp, String mood, long id) {
+        this.comment = comment;
+        this.timestamp = timestamp;
+        this.name = name;
+        this.mood = mood;
+        this.id = id;
+    }
+
+    public String getName(){
```

Comment:
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

`Before any open curly brace`

for this and all other methods:
getName() {